pkg/install/resource/injector_svc.go

```diff
@@ -0,0 +1,58 @@
+package resource
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	"github.com/datawire/ambassador/pkg/kates"
+	"github.com/telepresenceio/telepresence/v2/pkg/install"
+)
+
+type injectorSvc int
+
+var AgentInjectorSvc Instance = injectorSvc(0)
+
+func (ri injectorSvc) service(ctx context.Context) *kates.Service {
+	svc := new(kates.Service)
+	svc.TypeMeta = kates.TypeMeta{
+		Kind: "Service",
+	}
+	svc.ObjectMeta = kates.ObjectMeta{
+		Namespace: getScope(ctx).namespace,
+		Name:      install.AgentInjectorName,
+	}
+	return svc
+}
+
+func (ri injectorSvc) Create(ctx context.Context) error {
+	svc := ri.service(ctx)
+	svc.Spec = kates.ServiceSpec{
+		Type:     "ClusterIP",
+		Selector: getScope(ctx).tmSelector,
+		Ports: []kates.ServicePort{
+			{
+				Name: "https",
+				Port: 443,
+				TargetPort: kates.IntOrString{
+					Type:   intstr.String,
+					StrVal: "https",
+				},
+			},
+		},
+	}
```

Comment:
Is there a reason we don't just shove the port in to the existing manager service? Is there a reason to choose one approach over the other?

Reply:
The webhook mechanism is hardcoded to port 443 and I didn't want to dedicate that port for this since it uses a generated TLS that is useless for other purposes. The traffic-manager service has `ClusterIP: None` so it's headless and cannot do proxying and I wanted a service that proxies 443 to  8443 in the manager. I'm sure there are other ways to do this. Let's revisit.